src/endpoints/v1/schemas/events.json

```diff
@@ -46,80 +46,80 @@
     },
     "start.gt": {
       "type": "string",
-      "pattern": "^([0-9]{4})-[0-9]{2}-[0-9]{2}$"
+      "pattern": "^([0-9]{4})-[0-9]{2}-[0-9]{2}(T[0-9]{2}:[0-9]{2}:[0-9]{2}(Z)?((\\+|\\-)[0-9]{2,4})?)?$"
```

Comment:
It might help to add comments on each option to have a quick view on what we support? Either as a sentence or as an example(s)?